### PR TITLE
Remove runes and SoD proto values from core and UI

### PR DESCRIPTION
**Pull request by FelixPflaum**
Remove SoD related proto fields and values from core/common types. This includes runes, consumes and (world)buffs.
Also remove all core and UI core that used them.

Individual classes are untouched if core changes didn't require it. The HasRune() function simply returns false now until classes get a proper update.

### CODE REVIEW

File: proto/common.proto [hunk 1]
```diff
@@ -491,26 +465,15 @@ enum ZanzaBuff {
...
-enum AtalaiMojo {
-	AtalaiMojoUnknown = 0;
-	MojoOfWar = 1;
-	MojoOfForbiddenMagic = 2;
-	MojoOfLife = 3;
 }
 
 message MiscConsumes {
 	bool bogling_root = 1;
 	bool elixir_of_coalesced_regret = 2;
```
[the remark this answers is not in the document]
Comment (FelixPflaum) [reply proto/common.proto]: Done.

File: sim/core/buffs.go [hunk 2]
```diff
@@ -2371,9 +2146,6 @@ func ApplyWarchiefsBuffs(unit *Unit, buffs *proto.IndividualBuffs, isAlliance bo
 	if buffs.WarchiefsBlessing && isHorde {
 		ApplyWarchiefsBlessing(unit, "WarchiefsBuff")
 	}
```
[the remark this answers is not in the document]
Comment (FelixPflaum) [reply sim/core/buffs.go]: Removed the faction restriction.

File: ui/core/components/inputs/consumables.ts [hunk 3]
```diff
@@ -676,7 +549,6 @@ export const elixirOfCoalescedRegret = makeBooleanMiscConsumeInput({ actionId: (
 export const BoglingRoot = makeBooleanMiscConsumeInput({ actionId: () => ActionId.fromItemId(5206), fieldName: 'boglingRoot' });
 
 export const MISC_OFFENSIVE_CONSUMES_CONFIG: PickerStatOptions[] = [
-	{ config: Catnip, picker: IconPicker, stats: [] },
 	{ config: JujuFlurry, picker: IconPicker, stats: [Stat.StatAttackPower] },
 	{ config: elixirOfCoalescedRegret, picker: IconPicker, stats: [] },
```
[the remark this answers is not in the document]
Comment (FelixPflaum) [reply ui/core/components/inputs/consumables.ts]: Done.

**Review by ericbtaylorjr — APPROVED** [review]
Looks good, can do further cleanup later - but this is awesome since considerably less code muddying the view of getting things working. Good stuff.

**Pull request merged**